### DISCUSSION TITLE
Fanart Competition #2

### DISCUSSION THREAD

**Discussion by BirbSunflower** [sunflower-land/sunflower-land#2638] [Art] [answered]
For this competition, we encourage a wide range of art styles and techniques, not limited to pixel art. Feel free to explore your creativity and express yourself in any art style you choose!

Before submitting your artwork, we highly recommend that you read our fanart guidelines, which can be found here: 

https://docs.sunflower-land.com/contributing/competitions/fan-art-competition

## Theme 🎨

The theme for this month is: **Festa Junina** 🌽🔥👨‍🌾

> Festa Junina is the term used to describe the traditional festivities that happen at the beginning of the Brazilian winter in June. It is a month to commemorate some of the most famous saints for Catholics; namely Saint Anthony, Saint John the Baptist and Saint Peter. It is also a month to honor rain, the harvest season and marital union. Visitors celebrate from mid-June until the end of July and in some cities, festivities even continue until late August. Those attending will be treated to typical Brazilian food, will dress up like farmers, enjoy bonfires and dance the quadrilha. After Carnival, it is the second most important popular celebration in Brazilian culture.

We would love to see Sunflower Land's Version of Festa Junina! 😍

If you want to read more about Festa Junina, you can access this link: [Celebrating The Brazilian Harvest With Festa Junina](https://theculturetrip.com/south-america/brazil/articles/festa-junina-celebrating-the-brazilian-harvest/)

![image](https://github.com/BirbSunflower/SunflowerLand/assets/123522693/b57b315f-faeb-4087-bb4c-aa5fbf664683)

Show us your creativity by creating something related to the chosen theme! Whether it's a character that you love, a scene that captivates you, or a moment that leaves a lasting impression, we want to see what you can come up with. Let your imagination soar and share your unique perspective with us! ❤

## Requirements ✏️

- AI-generated art will not be accepted.
- The art should be original and not plagiarised.
- By submitting art, you acknowledge you are the original creator of the art and you give Thought Farm and the community the rights to use it freely in the game and marketing purposes indefinitely.
- You agree to the [terms and services](https://docs.sunflower-land.com/contributing/competitions/fan-art-competition#terms-and-conditions) of the game.

When submitting art, please include the following:

- Your Farm ID.
- Your Discord ID.
- A image of your art with good quality.
- A NPC that represents you. 
- A Google Drive link with proof of your creation. 
To verify the authenticity of your artwork, we kindly request that you provide proof of your creative process. This may include work-in-progress images, concept sketches, time-lapse videos, and, in the case of digital art, we also request the original files.

Please note that the inclusion of NPCs will not have any impact on the judging process of the competition. However, if you choose to provide an NPC, it will be displayed in-game alongside your art piece. If you do not include an NPC, a random one will be selected. Also, a Bumpkin version of your NPC would be appreciated, but not required. 

You can use this image to showcase your NPC:

![image](https://github.com/sunflower-land/sunflower-land/assets/123522693/81132ce6-4863-4be7-a3ca-b5f78434ee60)

## Rewards 🏆

The winner will receive 1000 SFL and a runner-up will receive 500 SFL, however not only will the winning artwork be showcased on the official Sunflower Land Game social media channels and in-game, but it will also be seen by our vibrant community of players from around the world.

We also want to emphasize that even if you don't win, your design may still have the chance to be selected in the future. Our team will carefully review all submissions and may reach out to designers whose work stands out for future collaboration opportunities. Sunflower Land is a community game and we love showcasing and including artwork from our players. Now is your chance to have your artwork immortalised in the #1 Community Web3 game.❤

## Deadline 📅

Artists have until **July 2** to submit their entries. After that, our art team will carefully review and analyze each submission to select the winners.

The announcement of the winners will take place during the first week of July.

So, what are you waiting for? Grab your brushes, pens, or digital tools and get creating! Show us what you're made of and join our community of talented artists in celebrating the vibrant world of Sunflower Land Game. We can't wait to see what you come up with!

Good luck to all participants! 🎉

Last Fanart Competition: https://github.com/sunflower-land/sunflower-land/discussions/2553

**Comment by LTNGx39** [comment]
### <h1> "Festa Junina!"

```
Farm ID: 161769
Discord: ncnlx39
```

My fan art for this occasion, I tried to give it as much focus as possible on the theme, I must mention that I didn't even know about this celebration until now, I hope you like it and it seems true to reality

Folder with files and tests: https://drive.google.com/drive/folders/1V-coPbWHkFuCyN9s3d0lPPGmn827MlfT?usp=sharing

![V19](https://github.com/sunflower-land/sunflower-land/assets/131133076/6bc15e9b-fdf9-4f30-8b70-357bc66d2479)

![V191](https://github.com/sunflower-land/sunflower-land/assets/131133076/86c9818a-a74a-47cb-9e7e-3ff89ff4541a)

**Comment by JcEii** [comment]
### Sunflower Land Festa Junina

**Discord:** Jceii
**Farm Id:** 15747

**Entry:**
![Festa-Junina-By-Jc-Eii-3](https://github.com/sunflower-land/sunflower-land/assets/105348036/131a44f3-0c5f-4866-9b60-908d4ad2008a)


**Google Drive:** https://drive.google.com/drive/folders/1tb0whEAdJpODMvGS977tvFNdbmCXeeOq?usp=drive_link


**My NPC:**
![My-NPC](https://github.com/sunflower-land/sunflower-land/assets/105348036/92afdb63-340c-4021-a52f-807d9018126b)
![jc_eii](https://github.com/sunflower-land/sunflower-land/assets/105348036/703d8732-4039-455e-84ce-1e601d331abe)

**Comment by Lvegas01** [comment]
Farm ID: 930
Discord: luisve

A new sunrise
The darkness has passed with the return of light now it's time to celebrate. What better than the Junina party !!!
![Festa Junina](https://github.com/sunflower-land/sunflower-land/assets/34526371/7aeabe38-99a8-4091-9555-dd9bafa5f4b1)

https://drive.google.com/file/d/1Im4vWv-jy9_cMnHHHATUl_v8gnZoIHed/view?usp=sharing

**Comment by Vergelsxtn** [comment, marked as answer]
### **Farm ID: 63**
**Discord: Vergelsxtn**
________
**Festa Junina in Sunflower Land**🌻🌽
![VERGELSXTN-SFL-COMP3](https://github.com/sunflower-land/sunflower-land/assets/99605039/d51558f4-4874-4289-bbcb-43189754346f)
___
**My Bumpkin & in-game NPC:**
Homeless guy | Bigtime edition 😂
![homeless-bumpkin2](https://github.com/sunflower-land/sunflower-land/assets/99605039/bcc11d98-13ea-4a3f-b51a-b94096852e2a)
![homeless-bumpkinFG2](https://github.com/sunflower-land/sunflower-land/assets/99605039/904fc4b7-871f-42ad-a19a-9bcdb2d3a8f9)
![template11-export](https://github.com/sunflower-land/sunflower-land/assets/99605039/023b6a2b-0e45-4f16-8dc4-5719d5fec600)
![template1](https://github.com/sunflower-land/sunflower-land/assets/99605039/c0dae1ea-0536-4584-9072-65b4dfd1620f)

Files:
https://drive.google.com/drive/folders/18a-EglR30rkXTUxZRt8M3yBYnw9F54SL?usp=sharing

**Comment by NycksonV** [comment]
'Farm ID: 136998'
'Discord: Nyck4168'

##Fã Art
'Festa Junina Pixel Art Giga'
![JuninaParty](https://github.com/sunflower-land/sunflower-land/assets/118012559/d45337cb-0af2-458a-96db-f2a61dac4862)

##

## Drive
https://drive.google.com/drive/folders/1kBbubfWftnnMut3PQOxW-8I215w2rgIf?usp=sharing

##

## Poro little Bumpkin
'This is me  u,u'
![Porinho](https://github.com/sunflower-land/sunflower-land/assets/118012559/289c37e7-3368-423a-8862-bff3b1f4e836)

| Sprite |
| --- |
![Porinho](https://github.com/sunflower-land/sunflower-land/assets/118012559/f0ed821a-1115-47cd-9eb8-95d2aa91186f)

##

## Sprite  ![Poro0](https://github.com/sunflower-land/sunflower-land/assets/118012559/db455ad6-eeff-4df9-a6e4-81093bd3d1f3)

##
